src/codeu/chat/client/Controller.java

```diff
@@ -33,26 +30,26 @@
 
   private final static Logger.Log LOG = Logger.newLog(Controller.class);
 
-  private final ConnectionSource source;
+  private final Connection source;
```

Comment:
You're probably sick of reading this, but I'd add comments to class-level variables like this. JavaDoc is your friend.